src/lib/collectors/shared/collector.ts

```diff
@@ -0,0 +1,733 @@
+/**
+ * @fileoverview Collector that uses the Chrome Debugging protocol to
```

Comment:
Let's rename the file to remote-debugging-collector or something similar. Just collector isn't right as it will not be shared by the others.